CHANGELOG.md

```diff
@@ -20,6 +20,8 @@ way to update this template, but currently, we follow a pattern:
   timing of the complete transition [#142](https://github.com/sharetribe/web-template/pull/142)
 - [fix] Email templates: fix layout-breaking condition in purchase templates.
   [141](https://github.com/sharetribe/web-template/pull/141)
+- [fix] Fix indentation in default transaction process email templates.
+  [#140](https://github.com/sharetribe/web-template/pull/140)
```

Comment:
@lyyder We try to follow the merge order here - so, this should have been the first one on the list.
That being said, no need to change it.